browser/brave_vpn/brave_vpn_service_factory.cc

```diff
@@ -60,8 +65,9 @@ KeyedService* BraveVpnServiceFactory::BuildServiceInstanceFor(
       default_storage_partition->GetURLLoaderFactoryForBrowserProcess();
 
 #if defined(OS_WIN) || defined(OS_MAC)
-  return new BraveVpnServiceDesktop(shared_url_loader_factory,
-                                    user_prefs::UserPrefs::Get(context));
+  return new BraveVpnServiceDesktop(
+      shared_url_loader_factory, user_prefs::UserPrefs::Get(context),
+      skus::SkusServiceFactory::GetForContextPrivate(context));
```

Comment:
call `skus::SkusServiceFactory::GetForContext` and pass in the mojo interface here

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/f1fd81200a1006b5c0971b6805968acf4934c79a - thanks for the help